test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -948,7 +949,7 @@ public void TestMulticlassEnsembleCombiner()
                             predGetters[i](ref preds[i]);
                         }
                         if (scores.All(s => !float.IsNaN(s)))
-                            CompareNumbersWithTolerance(score, scores.Sum() / predCount);
+                            CompareNumbersWithTolerance(score, scores.Sum() / predCount, digitsOfPrecision: 5);
```

Comment:
>digitsOfPrecision [](start = 89, length = 17)

why we use less precise here?

Reply:
The default is 7 digits. All tests are not equally precise. 

---
In reply to: [377899328](https://github.com/dotnet/machinelearning/pull/4823#discussion_r377899328) [](ancestors = 377899328)